cmd/server/main.go

```diff
@@ -83,13 +90,28 @@ func runService(c *cli.Context) error {
 		return err
 	}
 
-	rc, err := redis.GetRedisClient(conf.Redis)
+	rc, err := lkredis.GetRedisClient(conf.Redis)
 	if err != nil {
 		return err
 	}
 
-	rpcServer := egress.NewRedisRPCServer(rc)
-	svc := service.NewService(conf, rpcServer)
+	ctx := context.Background()
+	previous, err := rc.Get(ctx, ev.EgressVersionKey).Result()
```

Comment:
what happens when multiple egress instances are running? like as part of a rolling deploy?

Reply:
I think this should be ok with redis? I could add locking, but for that we just use a SetNX and check which one got the write